src/main/scala/edu/twitter/model/service/ModelService.scala

```diff
@@ -36,7 +36,7 @@ class ModelService(genericModelBuilder: GenericModelBuilder)
   def start(): Unit = {
     val model = genericModelBuilder.build()
     val route: Route =
-      path("classify") {
+      path(s"${model.name}" / "classify") {
```

Comment:
I believe you should pass a list of GenericModelBuilder to the constructor of this class, then you could keep a map between the model and its name and whenever a request is made you can get the model by its name and respond with the label.

Reply:
I guess if you need multiple models you could simply create multiple instances of `modelService` giving each instance its own `builder`. this will start multiple models for you.

Reply:
I was mainly asking for why rather than how, I thought it might be better to have one instance of the modelService that handle all the requests but anyway it doesn't really matter. Thanks 👍